### PR TITLE
Getting started with classes and objects in C++

### LINKED ISSUE #992: Getting started with classes and objects in C++
### Brief Summary: 
C++ is one of the Object-oriented programming (OOP) language. The limitations of C language are evident when a software project is too large to be handled. This problem was addressed by adding several extensions to the C language. The most important addition was the concept of class. In this article, we will take a look at classes and objects in depth.

### Key Takeaways: 
By the end of this tutorial, the reader will be able to:
1. Know what a class and object is
2. Create a class
3. Create objects
4. Use objects as function arguments

### References: 
N/A

### PR DESCRIPTION
**Pull request by Dawe-7**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #992

### CODE REVIEW

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written article.

File: articles/getting-started-with-classes-and-objects-in-c++/index.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+### Introduction
+The C language limitations are evident when a software project is too large to manage. This problem was addressed by adding several extensions to the C language. The main incorporation was the concept of class. The most significant aspect of C++ classes is that it promotes object-oriented programming which treats data as a critical element.  This article goes through the creation of classes, objects, and objects as function arguments.
+
+#### Prerequisites
+To follow this article, you’ll need to have:
+- [Codeblocks IDE](http://www.codeblocks.org/downloads) to run the code.
+- A basic understanding of C++ language.
+- A basic understanding of functions.
+
+#### What is a Class?
```
[the remark this answers is not in the document]
Comment (Dawe-7) [reply articles/getting-started-with-classes-and-objects-in-c++/index.md:10]: @lalith1403 could you explain more on this in order to make the change appropriately.

[the remark this answers is not in the document]
Comment (lalith1403) [reply articles/getting-started-with-classes-and-objects-in-c++/index.md:10]: I meant, don't make the headings and subheadings title cased. Instead, keep the first word of the headings capital.

For example:

What is a class?

[the remark this answers is not in the document]
Comment (Dawe-7) [reply articles/getting-started-with-classes-and-objects-in-c++/index.md:10]: Thanks for the clarification.

**Comment by Dawe-7** [comment]
@lalith1403 I've made the requested changes, kindly review.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**